src/widgets.js

```diff
@@ -749,8 +749,13 @@ define([
 
     function getMessages(mug, path) {
         var $messages = $(),
-            seen = {};
+            seen = {},
+            usesRichText = mug.form.vellum.opts().features.rich_text;
         mug.messages.each(path, function (msg) {
+            if (usesRichText && msg.key === "logic-bad-path-warning") {
```

Comment:
Thought about this some more, and I'm inclined to just revert the changes that move warnings to popovers, don't think we thoroughly thought through that change.

@amsagoff it's awkward to move the "Unknown question" warnings into popovers because that same warning appears for certain non-bubbling expressions (specifically, absolute paths that start with something other than `/data/`). So right now, on production, you can get something like this:

![pasted image at 2016_06_17 17_06](https://cloud.githubusercontent.com/assets/1486591/16168893/21ef57f2-34e7-11e6-8034-bcf0c5998a8d.png)

Seems weird to me that we'd move the `#form/question2` part of that error into a popover and leave the rest of it as-is. For the moment, I reverted the popover changes, so all unknown question errors will keep displaying in the yellow box, but let me know if you disagree.